app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
+  # before_action :configure_sign_up_params, only: [:create]
+  # before_action :configure_account_update_params, only: [:update]
+
+  # GET /resource/sign_up
+  # def new
+  #   super
+  # end
+
+  # POST /resource
+  # def create
+  #   super
+  # end
+
+  # GET /resource/edit
+  # def edit
+  #   super
+  # end
+
+  # PUT /resource
+  def update
+    # Call the parent Devise method to perform the default update action
+    super do |resource|
+      # If the new password and confirmation fields are blank, add an error
```

Comment:
Is it your own comments? You need them?

Reply:
I was needed in them first time, yes, it redundant